CodeEdit/NavigatorSidebar/ProjectNavigator/OutlineView/OutlineView.swift

```diff
@@ -38,6 +41,32 @@ struct OutlineView: NSViewControllerRepresentable {
         return
     }
 
+    func makeCoordinator() -> Coordinator {
+        return Coordinator(workspace)
+    }
+
+    class Coordinator: NSObject {
+        init(_ workspace: WorkspaceDocument) {
+            self.workspace = workspace
+            super.init()
+
+            listener = workspace.listenerModel.$highlightedFileItem.sink(receiveValue: { [weak self] fileItem in
```

Comment:
Move `sink` to new line for better readability.

```swift
listener = workspace.listenerModel.$highlightedFileItem
    .sink(receiveValue: { [weak self] fileItem in
        …
    })
```

Reply:
Got it, pushed that change.